src/transitions/Slide.js

```diff
@@ -75,12 +83,19 @@ export default class Slide extends Component {
      * Callback fired when the component has exited.
      */
     onExited: PropTypes.func, // eslint-disable-line react/sort-prop-types
+    /**
+     * If `false`, let the child element slide in from outside the viewport.
+     * Set to `true` to let the child element slide in from outside its own
+     * bounding box and position.
+     */
```

Comment:
We are using the pattern: "If `true` the child element slides in from outside its own bounding box and position"

Not sure how best to address the default case - perhaps follow the above with: "; by default, the child element slide in from outside the viewport."?